.lintstagedrc

```diff
@@ -1,4 +1,4 @@
 {
-  "*.js": "yarn lint:js",
-  "*.jsx": "yarn lint:jsx"
+  "*.js": ["npm run lint:js"],
```

Comment:
Thanks for changing this, I use Yarn so my bad. Why do we have square brackets though? Also, I can't see any config, script or package for style-lint. Did you read the SCSS story properly?

Reply:
I changed that...Its meant for multiple commands. but since we have added it to the pre-commit hook, it isn't necessary